torch/_dynamo/symbolic_convert.py

```diff
@@ -733,15 +733,56 @@ def step(self):
         assert not self.output.output_instructions
         assert self.checkpoint is not None
         continue_inst, state = self.checkpoint
+
+        cannot_restore_entry = next(
+            entry for entry in reversed(self.block_stack) if not entry.can_restore()
+        )
+        if cannot_restore_entry:
+            cannot_restore_target = cannot_restore_entry.target
+            insts = self.instructions[self.instruction_pointer :]
+            last_jump_forward = None
+            in_active_region = False
+            for inst in insts:
+                if inst == cannot_restore_target:
+                    in_active_region = True
+                if in_active_region and inst.opname == "JUMP_FORWARD":
+                    last_jump_forward = inst
+
         self.restore_graphstate(state)
         self.output.compile_subgraph(
             self,
+            # TODO: not sure if this needs to be changed if we are going to resume_at
             partial_convert=True,
```

Comment:
This part has to do with whether or not we setup block stacks... I believe the answer is yes? We do want partial convert?

Reply:
Yes, we do, because we will continue into the middle of the block.